### PR TITLE
Added DataportenApi

### PR DESCRIPTION
**Pull request by xibriz**
Dataporten is a national autentication provider in Norway used by many public companies and educational institusions.

### CODE REVIEW

File: scribejava-apis/src/main/java/com/github/scribejava/apis/DataportenApi.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    private static class InstanceHolder {
+        private static final DataportenApi INSTANCE = new DataportenApi();
+    }
+
+    public static DataportenApi instance() {
+        return InstanceHolder.INSTANCE;
+    }
+
+    @Override
+    public Verb getAccessTokenVerb() {
```
Comment (kullfar) [inline scribejava-apis/src/main/java/com/github/scribejava/apis/DataportenApi.java]: POST is default, no need to override

File: scribejava-apis/src/main/java/com/github/scribejava/apis/DataportenApi.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    @Override
+    protected String getAuthorizationBaseUrl() {
+        return "https://auth.dataporten.no/oauth/authorization";
+    }
+
+    @Override
+    public TokenExtractor<OAuth2AccessToken> getAccessTokenExtractor() {
+        return OAuth2AccessTokenJsonExtractor.instance();
```
Comment (kullfar) [inline scribejava-apis/src/main/java/com/github/scribejava/apis/DataportenApi.java]: this is default as well

**Comment by xibriz** [comment]
@kullfar Do you want me to make the changes and do a new PR?

**Comment by kullfar** [comment]
No, changes are minor. I'll make changes myself while will be merging Pull Request to the master.
But I have some troubles, running the Example. I've wrote to your email.

**Comment by xibriz** [comment]
I have replied. Since Dataporten is used by so many different organization it can be a little confusing, but you want the "Feide test users" listed under "Alternative login providers for non-educational users".

**Comment by kullfar** [comment]
well done, I have merged it to the master

**Comment by kullfar** [comment]
Thx again!

**Comment by xibriz** [comment]
Thank you good sir :)

**Pull request merged**